src/utils/validatorUtils.ts

```diff
@@ -220,3 +220,10 @@ function getParameterTagSupport() {
     'maxDate',
   ];
 }
+
+function removeSurroundingQuotes(str: string) {
+  if (str.startsWith('`') && str.endsWith('`')) {
```

Comment:
I really like this approach. My only remaining question is whether we should unquote triple quotes aswell which seems to be a legal way of escaping, too (?)

Reply:
If it's legal in JSDoc, we should probably support it as well. For pattern specifically it's not a big deal, but if this gets added for other tags it might be needed. I'll add it real quick